homeassistant/components/google/__init__.py

```diff
@@ -115,6 +115,7 @@ def scope(self) -> str:
         vol.Optional(CONF_OFFSET): cv.string,
         vol.Optional(CONF_SEARCH): cv.string,
         vol.Optional(CONF_TRACK): cv.boolean,
+        vol.Optional("max_results"): vol.All(cv.deprecated),  # Now unused
```

Comment:
`vol.All` needs to wrap the whole dict schema and `cv.deprecated` passed next to the existing dict schema. Specify what key is deprecated in by calling `cv.deprecated("max_results")`.

Reply:
I see. Done.